app/server/api/organisation_api.py

```diff
@@ -205,6 +207,46 @@ def get(self):
         }
         return make_response(jsonify(response_object)), 200
 
+class AttributeMapAPI(MethodView):
```

Comment:
I might be missing it, but I think this API is inaccessible-- there's no routing rules.

Reply:
Doiiii my bad!